index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <h1 class="main">Hello Microverse!</h1>
```

Comment:
- [ ] [OPTIONAL] I would like to recommend you change your h1 tag class name class="main" main is an HTML tag name and it can be confusing to use it as a CSS class name. Please change it for semantic reason and also it is a good practice :smile: